emcc.py

```diff
@@ -221,8 +221,6 @@ def base64_encode(b):
 
 class EmccOptions(object):
   def __init__(self):
-    self.opt_level = 0
-    self.debug_level = 0
     self.shrink_level = 0
```

Comment:
if we are getting rid of opt_level and debug_level, why not shrink_level too?

Reply:
Or I guess put another way, why is shrink_level different that it's here instead of on shared.Settings?

Reply:
Good point, we should refactor that as well. Done.